guides/lando-phpstorm.md

```diff
@@ -64,44 +90,31 @@ For performance reasons, it can be a negative to always have Xdebug enabled. Bes
 Xdebug extension but leave Xdebug disabled (as is the case with `pantheon` framework with `config: xdebug: false`).
 One option to do so is to use tooling such as:
 
-### Toggling for Apache
 ```yaml
+config:
+  # Set Xdebug off by default. We use the tooling below to turn it on as needed.
+  xdebug: false
+
 services:
   appserver:
-    xdebug: true
-    build_as_root:
-      - rm -f /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini && /etc/init.d/apache2 reload
-tooling:
-  xdebug-on:
-    service: appserver
-    description: Enable xdebug for Apache.
-    cmd: rm -f /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini && docker-php-ext-enable xdebug && /etc/init.d/apache2 reload && echo "Xdebug enabled"
-    user: root
+    overrides:
+      environment:
+        XDEBUG_MODE: 'debug,develop'
 
-  xdebug-off:
-    service: appserver
-    description: Disable xdebug for Apache.
-    cmd: rm -f /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini && /etc/init.d/apache2 reload && echo "Xdebug disabled"
-    user: root
-```
-                                             
-### Toggling for nginx
```

Comment:
Please don't take away the nginx version.  We use nginx. 😄   (Unless there's a unified version we can offer, which would be even better.)

Reply:
My update does both!